### ISSUE TITLE
[BORKED]drown_command error on Survival

### ISSUE THREAD

**Issue by Eutherin** [Adriftus-Studios/network-script-data#152]
> **Error Message**:

`Missing player for player tag.`

> **Script Information**:

Script Name:  `drown_command`

Script Reference:  [`survival/./repo_link/Commands/drown.dsc`](https://github.com/Adriftus-Studios/network-script-data/blob/master/denizen_scripts/survival/repo_link/Commands/drown.dsc#L10)

Script Line: 10

Caused by this:

![image](https://user-images.githubusercontent.com/25861695/97160054-089cca80-1739-11eb-8d98-8b603d990edf.png)
![image](https://user-images.githubusercontent.com/25861695/97160065-0b97bb00-1739-11eb-8753-b01acaffb233.png)

**Comment by BehrRiley** [comment]
This actually appears to be directly associated to the `execute` legacy discord command we have, it sometimes doesn't support quotation-wrapped arguments as single arguments. This will be resolved with a re-write for that command

**Comment by Eutherin** [comment]
What do we need to do to fix that command? it uses the same syntax as heal and feed

**Comment by BehrRiley** [comment]
The problem isn't the `drown` command - that's an internal error with the Discord `/ex` command we have set up that doesn't work properly

**Comment by Ph4i1ur3** [comment]
Or... maybe because `<player>` isn't valid when executing from console...?

**Comment by BehrRiley** [comment]
> Or... maybe because `<player>` isn't valid when executing from console...?

No. Referencing the script, the only time `<player>` is used is here:
https://github.com/Adriftus-Studios/network-script-data/blob/dd59c6df6732df0619667a697c42fbf51b071e98/denizen_scripts/survival/repo_link/Commands/drown.dsc#L10
which is only ran if:
https://github.com/Adriftus-Studios/network-script-data/blob/dd59c6df6732df0619667a697c42fbf51b071e98/denizen_scripts/survival/repo_link/Commands/drown.dsc#L9
`<context.args.is_empty>` is false; so this block never runs.

Per our `execute` command for Discord,
`/ex survival execute as_server "drown eutherin"` is executing as the server `Survival`, and is executing `as_server` the command: `drown eutherin`. `drown` is the command, and `eutherin` is the first and only argument. elsewise, the second block would run instead:
```yml
    - else if <context.args.size> == 1:
        - define User <context.args.first>
        - inject Player_Verification
```
indicating the `User` be defined as a valid player after being ran through the player verification. 

the problem is that [`[The execute command]`](https://github.com/Adriftus-Studios/network-script-data/blob/master/denizen_scripts/relay/repo_link/tasks/ex.dsc) does not support quoted arguments, which is why `<context.args.is_empty>` ran `true` and ran the block attempting to define `User` as `<player>`, which is invalid from Discord when there is no player attached to the queue.

...Does raise a good idea for #38 attaching the queue to the player running the command from discord if they have a linked Discord account, though. I'm adding that to it as a feature.
